tasks/public_goods_task.md

```diff
@@ -0,0 +1,43 @@
+# Summary
+> Players contribute some money to a public pool/box, which is totalled and multiplied by some factor, then divided equally amongst the players.
+
+# References
+> Main article: https://en.wikipedia.org/wiki/Public_goods_game
+> 
+> More info: Tomassini, M., &amp; Antonioni, A. (2020). Public goods games on coevolving social network models. _Frontiers in Physics_ https://doi.org/10.3389/fphy.2020.00058
+> 
+> Dong, Y., Zhang, B. & Tao, Y. The dynamics of human behavior in the public goods game with institutional incentives. _Sci Rep 6, 28809_ (2016). https://doi.org/10.1038/srep28809
+
+# Stimuli
+## The visual components
+> No visual components besides given materials: coins, bills, etc. and private receptacle to place money inside.
+
+## Materials for alternative versions of the experiment 
+> Multiple rounds: Same materials as the original version, but multiple rounds are played instead of one.
+> 
+> Transparent version: A see-through box can be used so participants know how much the previous player put in. Alternatively, players can be told how much the previous players put in.
+> 
+> Punishment/awards: A player can choose to punish "free-riders", i.e. those who don't contribute any money, by imposing a fine. Conversely, a player may choose to reward the highest contributer(s). Punishments and rewards are taken out of the player's own tokens.
+
+# Procedure
+## Steps
+> Players receive tokens.
+> 
+> One by one, players place as many or as little tokens as they wish into a private box. Players keep any tokens they did not give.
+> 
+> Tokens are counted up and multiplied by some factor, then divided equally between the players.
+> 
+> (The game can also be played on a computer program, doesn't need to have physical tokens and boxes.)
+
+## Roles 
+> No roles.
+
+## Instructions
+> See Steps.
+
+# Criteria
+## Performance calculation
+> Participants are evaluated by how many tokens they each give to the public pool.
+
+## Incentives
+> Participants do not receive incentives to participate.
```

Comment:
I wonder if this should be something like incentives depend on the researchers choice since there wasn't a specific study described in the article